app/styles/components/_button-group.scss

```diff
@@ -1,9 +1,12 @@
 // A group of buttons
 .BtnGroups {
-	padding-bottom: 2rem;
+	margin-bottom: 2rem;
 	display: flex;
 	flex-wrap: wrap;
 }
+.BtnGroups--noPaddingButton {
```

Comment:
does this do anything since we moved it from padding to margin?

Reply:
don't think so